tests/test_opds.py

```diff
@@ -549,6 +541,39 @@ def test_adobe_id_tags_when_vendor_id_configured(self):
         self._db.delete(setting)
         assert [] == self.annotator.adobe_id_tags("new identifier")
 
+    def test_lcp_acquisition_link_contains_hashed_passphrase(self):
+        [pool] = self.work.license_pools
+        identifier = pool.identifier
+        patron = self._patron()
+
+        # Setup LCP credentials
+        lcp_credential_factory = LCPCredentialFactory()
+        lcp_credential_factory.set_hashed_passphrase(
+            self._db, patron, "hashed password"
+        )
+
+        loan, ignore = pool.loan_to(patron, start=utc_now())
+        lcp_delivery_mechanism, ignore = DeliveryMechanism.lookup(
+            self._db, "text/html", DeliveryMechanism.LCP_DRM
+        )
+        other_delivery_mechanism, ignore = DeliveryMechanism.lookup(
+            self._db, "text/html", DeliveryMechanism.OVERDRIVE_DRM
+        )
+
+        # The fulfill link for non-LCP DRM does not include the hashed_passphrase tag.
+        link = self.annotator.fulfill_link(pool, loan, other_delivery_mechanism)
+        for child in link:
+            assert child.tag != "{http://readium.org/lcp-specs/ns}hashed_passphrase"
```

Comment:
It might make sense to put it in a separate variable in order to avoid duplication.